__mocks__/fileMock.js

```diff
@@ -0,0 +1 @@
+module.exports = 'sig-blk-en.svg';
```

Comment:
What is this doing? I haven't seen this before.

Reply:
this is because storybook does not like imports images for testing. This ignores the image issue. Omar helped me with this problem

Reply:
Cool, thanks for the explanation!